app/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CheckYourAnswersController.scala

```diff
@@ -71,20 +71,20 @@ class CheckYourAnswersController @Inject()(mcc: MessagesControllerComponents,
               case StartBusinessVerification(businessVerificationUrl) =>
                 Redirect(businessVerificationUrl)
               case JourneyCompleted(continueUrl) =>
-                if (journeyConfig.pageConfig.enableSautrCheck) auditService.auditSoleTraderJourney(journeyId)
-                else auditService.auditIndividualJourney(journeyId)
+                auditJourney(journeyId, journeyConfig)
                 Redirect(continueUrl + s"?journeyId=$journeyId")
               case SoleTraderDetailsMismatch(NinoNotFound) =>
-                if (journeyConfig.pageConfig.enableSautrCheck) auditService.auditSoleTraderJourney(journeyId)
-                else auditService.auditIndividualJourney(journeyId)
+                auditJourney(journeyId, journeyConfig)
                 Redirect(routes.DetailsNotFoundController.show(journeyId))
               case SoleTraderDetailsMismatch(_) =>
-                if (journeyConfig.pageConfig.enableSautrCheck) auditService.auditSoleTraderJourney(journeyId)
-                else auditService.auditIndividualJourney(journeyId)
+                auditJourney(journeyId, journeyConfig)
                 Redirect(routes.CannotConfirmBusinessErrorController.show(journeyId))
             }
         }
       }
   }
 
+  private def auditJourney(journeyId: String, journeyConfig: JourneyConfig)(implicit hc: HeaderCarrier): Future[Unit] =
+    if (journeyConfig.pageConfig.enableSautrCheck) auditService.auditSoleTraderJourney(journeyId, journeyConfig)
+    else auditService.auditIndividualJourney(journeyId, journeyConfig)
```

Comment:
Now that journey config is available in the audit method. This could be moved into the audit service, rather than having a private method.

Reply:
Done